src/auth/ducks/thunks.ts

```diff
@@ -10,6 +10,7 @@ export const login = (
   loginRequest: LoginRequest,
 ): UserAuthenticationThunkAction<void> => {
   return (dispatch, getState, { authClient, tokenService }): Promise<void> => {
+    // TODO: dispatch loading
```

Comment:
Are these (and by these I mean the other TODO too) going to be done in another PR or something?